src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -423,14 +431,22 @@ private function getClasses(array $entrypointProperties, array $classes): array
      */
     private function getProperty(PropertyMetadata $propertyMetadata, string $propertyName, string $prefixedShortName, string $shortName): array
     {
+        $propertyData = [
+            '@id' => $propertyMetadata->getIri() ?? "#$shortName/$propertyName",
+            '@type' => $propertyMetadata->isReadableLink() ? 'rdf:Property' : 'hydra:Link',
+            'rdfs:label' => $propertyName,
+            'domain' => $prefixedShortName,
+        ];
+
+        $type = $propertyMetadata->getType();
+
+        if (null !== $type && !$type->isCollection() && (null !== $className = $type->getClassName()) && $this->resourceClassResolver->isResourceClass($className)) {
```

Comment:
Is this part really necessary `(null !== $className = $type->getClassName()) && $this->resourceClassResolver->isResourceClass($className)`?
Even if it is not a resource, the max cardinality is 1 if it's not a collection isn't it?